util/setgroups_none.c

```diff
@@ -0,0 +1,60 @@
+/* We use non-POSIX functionality in this file. */
```

Comment:
This is the only file with non-POSIX functionality in libcperciva.  In scrypt, we isolated such code to `lib-platform/` instead of `lib`.  I figured that you wouldn't want to bother doing that here, but it's easy for me to tweak the build system if you do want that separation.